src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -32,3 +33,42 @@
 SCREENING_SCALE_UP_DIFFERENCE = (
     SCREENING_SCALE_UP_GOAL_COVERAGE - PROBABILITY_ATTENDING_SCREENING_START_MEAN
 )
+
+
+class __PregnancyOutcome(NamedTuple):
+    PARTIAL_TERM_OUTCOME = "partial_term"
+    LIVE_BIRTH_OUTCOME = "live_birth"
+    STILLBIRTH_OUTCOME = "stillbirth"
+    INVALID_OUTCOME = "invalid"  ## For sex of partial births
+
+
+PREGNANCY_OUTCOMES = __PregnancyOutcome()
+
+
+class _Durations(NamedTuple):
+    FULL_TERM_DAYS = 40 * 7
+    POSTPARTUM_DAYS = 6 * 7
+    PARTURITION_DAYS = 1 * 7
+    DETECTION_DAYS = 6 * 7
+    PARTIAL_TERM_DAYS = 24 * 7
+    INTERVENTION_DELAY_DAYS = 8 * 7
+
+
+DURATIONS = _Durations()
```

Comment:
Are these used? I think many (if not all) of these will not be relevant to this sim.

Reply:
Yes they get used to make the pregnancy duration choice.